Src/BootCamp.Chapter1/Program.cs

```diff
@@ -1,9 +1,27 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+
+namespace BootCamp.Chapter1
 {
     class Program
     {
         static void Main(string[] args)
         {
+           // int[] myArray = new int[] { 4, 8, 0, 1, 9, 3, 7, 2, 88, 54, 11, 18, 100, 6 };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4905609)